pagopa/pn-frontend
# Code modal on mobile - Fix/pn 7071

## Short description
Users that use Chrome on Android and try to insert a new contact, must click on every input to insert the otp. In this situation the automatic navigation doesn't work.

## List of changes proposed in this pull request
- Modified the CodeInput component

## How to test
Desktop -> check that there is no regression
Mobile -> check that the automatic navigation works on mobile
